content/en/logs/log_collection/nodejs.md

```diff
@@ -128,6 +128,53 @@ logs:
     sourcecategory: sourcecode
 ```
 
+## Aggentless Logging
+
+It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log to a file when your application is running on a machine that cannot be accessed.
+
+{{< tabs >}}
```

Comment:
Should we document API usage directly as well? A lot of users won't be using Winston, and we don't yet support other popular loggers out of the box.

Reply:
the API is already documented here: https://docs.datadoghq.com/api/?lang=bash#send-logs-over-http 
I'll add a link to it.

Reply:
added